debian/ceph-common.postrm

```diff
@@ -26,6 +26,8 @@ case "$1" in
     purge)
 	rm -rf /var/log/ceph 
 	rm -rf /etc/ceph
+	userdel -f ceph
```

Comment:
What about removing the files in the user's home directory along with the home directory itself using -r option ?
@liewegas Is it required to keep the files and home directory for future use ? Could you please suggest ?

I think the newer Debian Releases use deluser too.